src/user/user.service.ts

```diff
@@ -53,14 +53,13 @@ export class UserService {
     const userFromDb = await this.prisma.user.findUnique({
       where: { name },
     });
-
     if (userFromDb) {
       return userFromDb;
     }
 
     const hashedUserPassword = await this.hashPassword(password);
 
-    return this.prisma.user.create({
+    return await this.prisma.user.create({
```

Comment:
You usually don't need `return await` - https://eslint.org/docs/latest/rules/no-return-await

Reply:
Is it ok to squash commits while merging?